### PR TITLE
Enhancement / Bug - `newProject`

### PR DESCRIPTION
**Pull request by kiserd**
Fixed a few bugs in the `newProject` page:
- capacity field was being created as a string, converted form value to integer
- likes was not being added to the payload, manifesting in the absence of the likes field in Firebase (fixed this)
- census was being initialized as 1, but then incremented when the `projects_users` association was created (fixed it)

### CODE REVIEW

File: portfolio-project-matching-app/pages/test_dao.js [hunk 1]
```diff
@@ -26,101 +26,10 @@ import { User } from '../models/User'
...
-        // const field = 'dummy';
-        // const criteria = 'dummy';
-        // const ids = await readDocIdsByCriteria(coll, field, criteria);
-        // console.log('ids: ', ids);
-
-
+        // DELETE PROJECT
+        // const coll = 'projects';
+        // const id = 'aI0TV2UfGcUTAWIZLTQp';
+        // await deleteDocAndAssociations(coll, id);
```
Comment (kiserd) [inline portfolio-project-matching-app/pages/test_dao.js:32]: Just cleaned up my testing file a bit

File: portfolio-project-matching-app/pages/newProject.js [hunk 2]
```diff
@@ -93,9 +93,10 @@ const newProject = () => {
         // get open field value by examining capacity
         const openStatus = payload.capacity === 1 ? false : true;
         // build on payload object with values from form
-        // there has to be a better way to do this
         const newPayload = payload;
-        newPayload.census = 1;
+        newPayload.census = 0; // createAssociation will increment to 1
```
Comment (kiserd) [inline portfolio-project-matching-app/pages/newProject.js:97]: census was being incremented when the `projects_users` association was created, so initializing to 1 was redundant.

File: portfolio-project-matching-app/pages/newProject.js [hunk 3]
```diff
@@ -93,9 +93,10 @@ const newProject = () => {
...
+        newPayload.capacity = parseInt(newPayload.capacity, 10)
```
Comment (kiserd) [inline portfolio-project-matching-app/pages/newProject.js:98]: This was manifesting in a string in Firebase before the fix.

File: portfolio-project-matching-app/pages/newProject.js [hunk 4]
```diff
@@ -93,9 +93,10 @@ const newProject = () => {
...
+        newPayload.likes = 0;
```
Comment (kiserd) [inline portfolio-project-matching-app/pages/newProject.js:99]: I completely forgot to create the likes field in the original payload. Obviously, this resulted in some unexpected behavior when a user attempted to increment the likes via `browseProjects`

**Review by Josh-Kaiser-OSU — APPROVED** [review]

**Review by ylijokic — APPROVED** [review]

**Pull request merged**